frontend/src/components/exploreCourseList/ExploreCourseList.jsx

```diff
@@ -26,23 +64,86 @@ const ExploreCourseList = (props) => {
       </span>
       <div className="explore-course-list" ref={courseContainerRef}>
         {props.courses.map((course, index) => (
-          <article
+          <div
```

Comment:
Given we are iterating through a list of courses, this would probably a good breaking point to encapsulate this code to display a course into it's own component. This would make it easier to grok this file and not need to dig into more detail unless needed.